pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1367,6 +1369,17 @@ func (l *LibvirtDomainManager) GetGuestInfo() (v1.VirtualMachineInstanceGuestAge
 	return guestInfo, nil
 }
 
+// InterfacesStatus returns the interfaces Guest Agent reported
+func (l *LibvirtDomainManager) InterfacesStatus(domainInterfaces []api.Interface) []api.InterfaceStatus {
+	interfaces := l.agentData.GetInterfaceStatus()
+	return agentpoller.MergeAgentStatusesWithDomainData(domainInterfaces, interfaces)
+}
+
+// GetGuestOSInfo returns the Guest OS version and architecture
+func (l *LibvirtDomainManager) GetGuestOSInfo() api.GuestOSInfo {
```

Comment:
Didn't we dropped the `get` previously?

Reply:
On the other function `InterfacesStatus` not on this one, i can drop this one as well 

Referring to https://github.com/kubevirt/kubevirt/pull/5560#pullrequestreview-649883591
This is the only reason i didn't squash yet, i will squash at the end, as stated previously

Reply:
> I didn't squash yet so it will be easier to see what are the new changes, i will squash at the end, as stated previously

I do not know why I need to bother myself with what was before, when it had a bug.
It makes me harder to review.
If you got a request to split from another reviewer, then fine. Otherwise, I do not understand how is this helpful.

Reply:
It was helpful for me, i will squash as soon as possible